### PR TITLE
Added google doc

### PR DESCRIPTION
**Pull request by Karama17**
This is my homework.

### CODE REVIEW

File: hw03_karamad.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# TODO Do not edit this file directly! Instead, create a new file called
+#        hw03_username.py and copy this code into it!
+
```
Comment (sheggen) [inline hw03_karamad.py:3]: Delete the template text.

File: hw03_karamad.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+def drawSquare(bob, sz):
+    """Make turtle t draw a square with side sz."""
+    for _ in range(4):
+        bob.forward(sz)
+        bob.left(90)
+
+
+def window(bob):
+    drawSquare(bob, 80)
```
Comment (sheggen) [inline hw03_karamad.py:29]: All functions need a docstring

File: hw03_karamad.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+if __name__ == "__main__":
+    main()      # starts the program
+
+
+
+
+
+
+
```
Comment (sheggen) [inline hw03_karamad.py:135]: Remove the extra white space.

**Comment by Karama17** [comment]
@sheggen can you check it know

**Pull request closed without merging**